src/metabase/models/notification.clj

```diff
@@ -54,12 +121,19 @@
                           [:= :ns.event_name (u/qualified-name event-name)]
                           [:= :ns.type (u/qualified-name :notification-subscription/system-event)]]}))
 
-(defn create-notification!
+(mu/defn create-notification!
```

Comment:
You don't seem to have added any type annotations to go with this

Reply:
yea, I'm adding one in the next pr, but will remove that mu/ for now